x-pack/plugins/ml/public/components/rule_editor/scope_expression.js

```diff
@@ -94,7 +95,12 @@ export class ScopeExpression extends Component {
 
     return (
       <div style={POPOVER_STYLE}>
-        <EuiPopoverTitle>Is</EuiPopoverTitle>
+        <EuiPopoverTitle>
+          <FormattedMessage
+            id="xpack.ml.ruleEditor.scopeExpression.isTitle"
```

Comment:
This id should be changed to use `scopeFilterTypePopoverTitle` rather than `isTitle`